hatch/cli.py

```diff
@@ -16,8 +17,9 @@ def get_command(self, ctx, cmd_name):
 
 @click.group(cls=AliasedGroup, context_settings=CONTEXT_SETTINGS)
 @click.version_option()
-def hatch():
-    pass
+@click.pass_context
+def hatch(ctx):
+    ctx.obj = Project()
```

Comment:
I recently made some refactors to decrease startup times. Could we please only import/use a Project where it is needed?